src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -90,9 +93,17 @@ Status CoreWorkerPlasmaStoreProvider::Get(const std::vector<ObjectID> &ids,
     for (size_t i = 0; i < object_buffers.size(); i++) {
       if (object_buffers[i].data != nullptr) {
         const auto &object_id = unready_ids[i];
-        (*results)[unready[object_id]] =
-            std::make_shared<PlasmaBuffer>(object_buffers[i].data);
+        (*results)[unready[object_id]] = std::make_shared<RayObjectValue>(
+            std::make_shared<PlasmaBuffer>(object_buffers[i].data),
+            std::make_shared<PlasmaBuffer>(object_buffers[i].metadata));
         unready.erase(object_id);
+        // TODO (kfstorm): metadata should be structured.
+        std::string metadata = object_buffers[i].metadata->ToString();
+        for (auto &error_type : EnumValuesErrorType()) {
```

Comment:
missing the definition for `EnumValuesErrorType`?

Reply:
`EnumValuesErrorType` is generated by flatbuffer. PS: This part of code should not be in this PR. It will come with Java worker changes. I'll remove it.